### DISCUSSION TITLE
I can use help fixing this

### DISCUSSION THREAD

**Discussion by mbennett-labs** [nalanios/matrade-project#7] [General]
Hi! Apologies', I was just trying to add a note with the link to the OSWP testing Guide which can be quite helpful. Unfortunately I added another branch. I am trying to delete it but cannot seem to be able to . If you can delete that branch and make this link available I appreciate it:   
https://drive.google.com/file/d/1JmPBkFkV6JE03JN7-DQHn-fZa0kuiZxm/view?usp=sharing

**Comment by nalanios** [comment]
Hi @3canfreekit. Since you made the change in the new branch you created, all you have to do is create a new Pull Request. When you do that, you'll use the drop downs to select main for the base branch and your new branch as the compare branch. Then I'll review/approve it, and the branch will be deleted, with your changes in main :)
